CHANGES.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+### 0.1.7 2017-01-03
+
+* Update gltf-pipeline to 0.1.0-alpha9
+* Added command to generate documentation (npm run jsdoc)
```

Comment:
Since the README is both in the root folder and appended to doc/index.html one of the paths will be wrong, unless there in images in two places or it is somehow possible for jsdoc to edit the paths.